src/content/docs/mobile-monitoring/new-relic-mobile-android/get-started/new-relic-android-and-dt.mdx

```diff
@@ -0,0 +1,49 @@
+---
+title: New Relic Android mobile monitoring with distributed tracing
+metaDescription: This page describes how to set up Android distributed tracing.
+redirects:
+
+  - /docs/mobile-monitoring/new-relic-mobile-android/get-started/new-relic-mobile-and-dt
+---
+New Relic Android mobile monitoring agents support distributed tracing. This lets you see how your mobile app activity connects to related services.
+
+Watch this short video (approx. 2:15 minutes) to learn how to:
+
+* Find mobile distributed tracing data in New Relic.
+* Filter by standard and custom attributes.
+
+<Video id="tkAxeE5cT2Q" type="youtube" />
+
+## Requirements
```

Comment:
The anchors are missing.

Reply:
@urbiz-nr, good catch! I've fixed these!